core/src/main/java/org/apache/accumulo/core/client/impl/TabletServerBatchWriter.java

```diff
@@ -699,7 +701,16 @@ else if (Tables.getTableState(context.getInstance(), table) == TableState.OFFLIN
 
     }
 
-    void addMutations(MutationSet mutationsToSend) {
+    void queueMutations(final MutationSet mutationsToSend) {
+      queueThreadPool.submit(new Runnable() {
```

Comment:
> I didn't say there were multiple threads

Didn't intend to imply you said that.   Was comparing a thread pool with 1 thread to the following statement.

> You just have one thread always running, looking for mutations to add.